packages/rest/src/controllers/credentials/CredentialController.ts

```diff
@@ -97,6 +99,24 @@ export class CredentialController {
     }
   }
 
+  /**
+   * Creates a credential offer not bound to any existing connection
+   */
+  @Post('/offer-outofband-credential')
+  public async offerCredentialOutOfBand(
+    @Body()
+    offer: CredentialOfferTemp
+  ) {
+    const credential = await this.agent.credentials.createOutOfBandOffer(offer)
+
+    return {
+      message: `${this.agent.config.endpoints[0]}/?d_m=${JsonEncoder.toBase64URL(
+        credential.offerMessage.toJSON({ useLegacyDidSovPrefix: this.agent.config.useLegacyDidSovPrefix })
```

Comment:
Is the legacy did sov prefix also supported for the proof endpoint?

Reply:
I added it to the out of band proof request too. Also cleaned up the code a little.